### PR TITLE
NN-4278 add v1.2 ingress

### PR DESCRIPTION
**Pull request by timoj58**

### CODE REVIEW

File: helm_deploy/whereabouts-api/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ ingress:
   # live-1 is blue, live is green
   contextColour: green
   annotations:
-    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
```
Comment (sp-watson) [inline helm_deploy/whereabouts-api/values.yaml:11]: Doesn't this need to be kept at 100?

File: helm_deploy/whereabouts-api/templates/ingress-v1-2.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (sp-watson) [inline helm_deploy/whereabouts-api/templates/ingress-v1-2.yaml]: Not sure whether this will work - can't really review it

Comment (timoj58) [reply helm_deploy/whereabouts-api/templates/ingress-v1-2.yaml]: its just coped from the other ingress.  its same value

File: helm_deploy/whereabouts-api/values.yaml (hunk 1 above)
Comment (timoj58) [reply helm_deploy/whereabouts-api/values.yaml:11]: the old one is set to 0 (which is a valid value) and the new to 100 as per the guide

**Review by sp-watson — APPROVED** [review]

**Pull request merged**